hasher-matcher-actioner/hmalib/scripts/cli/run_api.py

```diff
@@ -60,7 +60,9 @@ def execute(self, tf_outputs: t.Dict) -> None:
         for k in fn_env_vars:
             os.environ[k] = fn_env_vars[k]
 
-        from hmalib.lambdas.api.api_root import app
+        from hmalib.lambdas.api.api_root import bottle_init_once
```

Comment:
nit: can this be at the toplevel? Code smell to have inline imports

If not, can you add comment?

Reply:
I'll add comment. In this case, it can't because of a second code smell which is that the lambdas read environment levels at module scope.